adapts jar paths in core MANIFEST as well as plugin MANIFESTs
--- a/freeplane/META-INF/MANIFEST.MF
+++ b/freeplane/META-INF/MANIFEST.MF
@@ -7,14 +7,15 @@
  lib/freeplaneeditor.jar,
  lib/freeplanemac.jar,
  lib/freeplaneosgi.jar,
- lib/commons-lang-2.6.jar,
- lib/commons-io-2.4.jar,
- lib/forms-1.2.1.jar,
- lib/gnu-regexp-1.1.4.jar,
- lib/SimplyHTML.jar,
- lib/jortho.jar,
- lib/idw-gpl.jar,
- lib/commons-codec-1.7.jar
+ ../../../java/commons-lang.jar,
+ ../../../java/commons-io.jar,
+ ../../../java/jgoodies-forms.jar,
+ ../../../java/jgoodies-common.jar,
+ ../../../java/gnu-regexp.jar,
+ ../../../java/SimplyHTML.jar,
+ ../../../java/jortho.jar,
+ ../../../java/idw.jar,
+ ../../../java/commons-codec.jar
 Bundle-Activator: org.freeplane.main.osgi.Activator
 Import-Package: org.osgi.framework,
  org.osgi.service.url;version="1.0.0"
--- a/freeplane_plugin_help/META-INF/MANIFEST.MF
+++ b/freeplane_plugin_help/META-INF/MANIFEST.MF
@@ -8,5 +8,5 @@
 Bundle-RequiredExecutionEnvironment: J2SE-1.5
 Require-Bundle: org.freeplane.core;bundle-version="1.0.1"
 Bundle-ClassPath: lib/plugin.jar,
- lib/jhall.jar
+ ../../../java/jhall.jar
 Export-Package: org.freeplane.plugin.help;uses:="org.freeplane.core.ui,org.freeplane.core.controller,org.osgi.framework"
--- a/freeplane_plugin_latex/META-INF/MANIFEST.MF
+++ b/freeplane_plugin_latex/META-INF/MANIFEST.MF
@@ -10,4 +10,4 @@
 Require-Bundle: org.freeplane.core;bundle-version="1.0.1",
  org.freeplane.plugin.script;bundle-version="1.0.1"
 Bundle-ClassPath: lib/plugin.jar,
- lib/jlatexmath.jar
+ ../../../java/jlatexmath.jar
--- a/freeplane_plugin_script/META-INF/MANIFEST.MF
+++ b/freeplane_plugin_script/META-INF/MANIFEST.MF
@@ -8,8 +8,8 @@
 Bundle-RequiredExecutionEnvironment: JavaSE-1.6
 Require-Bundle: org.freeplane.core;bundle-version="1.0.1"
 Bundle-ClassPath: lib/plugin.jar,
- lib/jsyntaxpane.jar,
- lib/groovy-all.jar
+ ../../../java/jsyntaxpane.jar,
+ ../../../java/groovy-all-2.x.jar
 Export-Package: jsyntaxpane,
  jsyntaxpane.actions,
  jsyntaxpane.actions.gui,
--- a/freeplane_plugin_svg/META-INF/MANIFEST.MF
+++ b/freeplane_plugin_svg/META-INF/MANIFEST.MF
@@ -7,27 +7,14 @@
 Import-Package: org.osgi.framework;version="1.3.0"
 Bundle-RequiredExecutionEnvironment: J2SE-1.5
 Require-Bundle: org.freeplane.core;bundle-version="1.0.1"
-Bundle-ClassPath: lib/batik-awt-util.jar,
- lib/batik-bridge.jar,
- lib/batik-css.jar,
- lib/batik-dom.jar,
- lib/batik-ext.jar,
- lib/batik-extension.jar,
- lib/batik-gui-util.jar,
- lib/batik-gvt.jar,
- lib/batik-parser.jar,
- lib/batik-script.jar,
- lib/batik-anim.jar,
- lib/batik-codec.jar,
- lib/batik-svg-dom.jar,
- lib/batik-svggen.jar,
- lib/batik-swing.jar,
- lib/batik-transcoder.jar,
- lib/batik-util.jar,
- lib/batik-xml.jar,
- lib/js.jar,
- lib/pdf-transcoder.jar,
- lib/xerces_2_5_0.jar,
- lib/xml-apis.jar,
+Bundle-ClassPath: ../../../java/batik-all.jar,
+ ../../../java/js.jar,
+ ../../../java/fop.jar,
+ ../../../java/avalon-framework.jar,
+ ../../../java/commons-logging.jar,
+ ../../../java/commons-io.jar,
+ ../../../java/xmlgraphics-commons.jar,
+ ../../../java/xercesImpl.jar,
+ ../../../java/xml-apis.jar,
  lib/plugin.jar,
- lib/xml-apis-ext.jar
+ ../../../java/xml-apis-ext.jar
--- a/freeplane_plugin_openmaps/META-INF/MANIFEST.MF
+++ b/freeplane_plugin_openmaps/META-INF/MANIFEST.MF
@@ -6,5 +6,5 @@
 Bundle-Activator: org.freeplane.plugin.openmaps.Activator
 Import-Package: org.osgi.framework;version="1.3.0"
 Bundle-RequiredExecutionEnvironment: J2SE-1.5
-Bundle-ClassPath: lib/JMapViewer.jar, lib/plugin.jar
+Bundle-ClassPath: ../../../java/jmapviewer.jar, lib/plugin.jar
 Require-Bundle: org.freeplane.core;bundle-version="1.0.1"
